### ISSUE TITLE
can't overwrite OSGi header for source bundle

### ISSUE THREAD

**Issue by missedone** [stempler/bnd-platform#32]
hi @stempler , 
I'm [OSGifying the TestNG 6.10](https://github.com/testng-team/testng-p2/blob/master/build.gradle), the P2 repo created successfully, but I can't install it to Eclipse due to error `Missing requirement: testng Sources 6.10.0.rel (org.testng.source 6.10.0.rel) requires 'package javax.xml.parsers 0.0.0' but it could not be found`

the "Import-Package" header of the TestaNG source bundle is below:
```
Import-Package: bsh;version="[2.0.0,3.0.0)";resolution:=optional,com.beu
 st.jcommander;version="[1.7.0,3.0.0)";resolution:=optional,com.beust.jc
 ommander.converters;version="[1.7.0,3.0.0)";resolution:=optional,com.go
 ogle.inject;version="[1.2,1.3)";resolution:=optional,junit.framework;ve
 rsion="[3.8.1, 5.0.0)";resolution:=optional,org.junit;resolution:=optio
 nal,org.junit.runner;resolution:=optional,org.junit.runner.manipulation
 ;resolution:=optional,org.junit.runner.notification;resolution:=optiona
 l,org.apache.tools.ant;version="[1.7.0, 2.0.0)";resolution:=optional,or
 g.apache.tools.ant.taskdefs;version="[1.7.0, 2.0.0)";resolution:=option
 al,org.apache.tools.ant.types;version="[1.7.0, 2.0.0)";resolution:=opti
 onal,org.apache.tools.ant.types.resources;version="[1.7.0, 2.0.0)";reso
 lution:=optional,org.apache.tools.ant.types.selectors;version="[1.7.0,
 2.0.0)";resolution:=optional,org.yaml.snakeyaml;version="[1.6,2.0)";res
 olution:=optional,org.yaml.snakeyaml.constructor;version="[1.6,2.0)";re
 solution:=optional,org.yaml.snakeyaml.nodes;version="[1.6,2.0)";resolut
 ion:=optional,javax.xml.parsers,javax.xml.xpath,org.w3c.dom,org.xml.sax
 ,org.xml.sax.helpers
```

I don't think we need "Import-Package" (or even "Export-Package") for a source bundle, so I tried to overwrite it, but it does not work:
```groovy
platform {
    override {
        optionalImport 'javax.*'
    }
    bundle "org.testng:testng:${version}", {
        bnd {
            optionalImport 'com.google.common.*'
            optionalImport 'org.w3c.*'
            optionalImport 'org.xml.*'
            version = versionWithQualifier
        }
        exclude module: 'ant'
    }
// Nick: trying to overwrite the "Import-Package" header for a source bundle
    bnd (group: 'org.testng', name: 'testng', version: version, classifier: 'sources') {
        instruction 'Import-Package', "!"
    }
    bnd (name: 'bsh') {
        optionalImport 'org.apache.bsf.*'
    }

    featureId 'org.testng.p2.feature'
    featureName 'TestNG P2 Feature'
    featureVersion versionWithQualifier
    featureProvider 'TestNG Team'

    categoryName 'TestNG P2 Libraries'
    categoryId 'org.testng.p2.libraries'

    useBndHashQualifiers false
    useFeatureHashQualifiers false
    defaultQualifier ''

    downloadsDir = new File(System.properties['user.home'], '.bnd-eclipse')
    eclipseMirror.linux.gtk.x86_64 = 'http://download.eclipse.org/eclipse/downloads/drops4/R-4.5.2-201602121500/eclipse-platform-4.5.2-linux-gtk-x86_64.tar.gz'
    eclipseMirror.macosx.cocoa.x86_64 = 'http://download.eclipse.org/eclipse/downloads/drops4/R-4.5.2-201602121500/eclipse-platform-4.5.2-macosx-cocoa-x86_64.tar.gz'
}
```

**Comment by missedone** [comment]
so in my case, i'd like to remove both header 'Import-Package' and 'Export-Package', looks like the existing logic does not work, https://github.com/stempler/bnd-platform/blob/master/src/main/groovy/org/standardout/gradle/plugin/platform/internal/util/bnd/BundleHelper.groovy#L71
I still see the 'Export-Package' header in the source bundle

**Comment by missedone** [comment]
fixed in PR #33
